### ISSUE TITLE
Qt 5.6 (WebEngine) Supports

### ISSUE THREAD

**Issue by Buzzroid** [tomahawk-player/tomahawk#527]
Qt 5.6 does not support WebKit any longer on which Tomahawk depends.
Any plan for Tomahawk to change from WebKit to WebEngine?

Regards,
James

**Comment by dschmidt** [comment]
It's not as easy as it might sound. QtWebKit supports synchronous function calls on which we depend to a certain degree. Also QtWebEngine can't be (cross)compiled with MinGW ... overall a pretty bad situation for us :-\

**Comment by justin-time** [comment]
There are some people who a working on a new QtWebKit:
https://github.com/annulen/webkit
http://lists.qt-project.org/pipermail/development/2016-June/026282.html

**Comment by dschmidt** [comment]
Yup, but that is also based on WebKit2 afaik - and that leaves us again with no synchronous calls

**Comment by a17r** [comment]
https://doc.qt.io/qt-5/qtwebenginewidgets-qtwebkitportingguide.html

**Comment by hugolm84** [comment]
Yes, I have been able to build and run using QtWebkit from https://github.com/annulen/webkit (qtwebkit-dev-wip) on OSx.  I have cross compiled it using mxe toolchain (https://mxe.cc). I managed to play tracks from soundcloud etc. 

https://github.com/tomahawk-player/tomahawk/blob/f69c8bc708783182fa4a29b044a15fb91e6984ea/Docker/Dockerfile#L207

**Comment by a17r** [comment]
We've been doing that for 2.5 years in Gentoo. Regardless, this dependency will be the reason that tomahawk will be dropped from our package repositories eventually.
